bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/NavigationImpl.java

```diff
@@ -85,6 +89,7 @@
     private String navigationRootPage;
     private List<NavigationItem> items;
     private boolean skipNavigationRoot;
+    private int rootLevel;
```

Comment:
I'd call it `structureStart` to be consistent with the existing `structureDepth` config which is also relative to the navigation root.

Reply:
Done in https://github.com/adobe/aem-core-wcm-components/pull/836/commits/a051a2f1675345cb3ac0ae1ae4d6656170c1d947